mitosheet/src/components/taskpanes/AITransformation/AITransformationTaskpane.tsx

```diff
@@ -0,0 +1,360 @@
+                        }}
+                        height='small'
+                        autoFocus
+                        darkBorder
+                    />
+                    <TextButton
+                        onClick={async () => {
+                            const randomHint = HINTS[Math.floor(Math.random() * HINTS.length)];
+                            setPromptState(prevPromptState => {return {...prevPromptState, loading: true, hint: randomHint}})
+
+                            const currentSelection = getSelectionForCompletion(props.uiState, props.gridState, props.sheetDataArray);
+                            const completionOrError = await props.mitoAPI.getAICompletion(promptState.userInput, currentSelection);
+                            if (completionOrError !== undefined && 'completion' in completionOrError) {
+                                const newParams = {...completionOrError, edited_completion: completionOrError.completion};
+                                setParams(newParams);
+                                setPreviousParams(prevPreviousParams => {
+                                    const newPreviousParams = [...prevPreviousParams];
+                                    newPreviousParams.push(newParams);
+                                    return newPreviousParams;
+                                })
+                                setOpenSections(prevOpenSections => {return {...prevOpenSections, 'Examples': false, 'Result': false}})
+                            } else if (completionOrError !== undefined && 'error' in completionOrError){
+                                setPromptState(prevPromptState => {
+                                    return {
+                                        ...prevPromptState,
+                                        error: completionOrError.error
+                                    }
+                                })
+                            }
+
+                            setPromptState(prevPromptState => {return {...prevPromptState, loading: false}})
+                        }}
+                        disabled={promptState.userInput.length === 0 || promptState.loading}
+                        variant='dark'
+                    >
+                        Generate Code
+                    </TextButton>
+                    {promptState.error !== undefined && 
+                        <p className="text-color-error">{promptState.error}</p>
+                    }
+                    {promptState.loading && 
+                        <p className="text-subtext-1">Generating code... {promptState.hint !== undefined ? `Hint: ${promptState.hint}` : ''}</p>
+                    }
+                    <Spacer px={10}/>
+                    <TextArea 
+                        value={params.edited_completion} 
+                        placeholder='df["column"] = True'
```

Comment:
IMO something like "See generated code here... " is a better placeholder because it's a bit confusing when going from actual generated code to this that is also code. It kinda seems like there is still code written. 

Either that or make the placeholder text lighter so it's more obvious.